CLTargs.rst

```diff
@@ -0,0 +1,56 @@
+positional arguments:
```

Comment:
This is just a 'placeholder' file for now, where i dumped out all the stuff from doing processCcd.py --help (i.e. the standard CLT args) into this file, but it will be cleaned up/replaced by a more correct file later.